cmd/main.go

```diff
@@ -81,8 +82,9 @@ func main() {
 	runtimeWebhookServerOpts := server.NewServerOptions()
 
 	globalOptions := options.NewGlobalOptions()
+	helmLifecycleOptions := lifecycleoptions.New()
```

Comment:
Why not put this in `globalOptions`?

Reply:
the name globaloptions... is so misleading. i can put it in there, but with the name i thought it was used in more than just the lifecycle package